### PR TITLE
Created primenumbers_till_n_nestedforloop.cpp

### LINKED ISSUE #478: Prime Number till N ( Algorithm )
In this algorithm i will try to find the number of prime number present till n and give a count with each number !! Will start working on it as soon as i get assigned

### PR DESCRIPTION
**Pull request by rohank2502**
closes #478 

I have  added one file in code/c++

Name of file is primenumbers_till_n_nestedforloop.cpp

### CODE REVIEW

File: Code/C++/primenumbers_till_n_nestedforloop.cpp [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+97
+Total Prime Number till N are : 25
+
+
+*/
+
+/*
+
+Time Complexity As we are using two nested loop the time complexity of this problem is O(N^2)
+
```
Comment (rudrakshi99) [inline Code/C++/primenumbers_till_n_nestedforloop.cpp:122]: Add space complexity also.

Comment (rohank2502) [reply Code/C++/primenumbers_till_n_nestedforloop.cpp:122]: Ok !!

File: Code/C++/primenumbers_till_n_nestedforloop.cpp [hunk 2]
```diff
@@ -121,3 +121,7 @@ Total Prime Number till N are : 25
 Time Complexity As we are using two nested loop the time complexity of this problem is O(N^2)
 
 */
+/*
+
+Space Complexity In this problem we have  declared 4 variables n,count,num & i so if we assume each varible takes 1 unit then space complexity is 4
```
Comment (rudrakshi99) [inline Code/C++/primenumbers_till_n_nestedforloop.cpp:124]: Do not write the statement. Just mention in terms of asymptotic notation.

File: Code/C++/primenumbers_till_n_nestedforloop.cpp [hunk 3]
```diff
@@ -121,3 +121,7 @@ Total Prime Number till N are : 25
 Time Complexity As we are using two nested loop the time complexity of this problem is O(N^2)
```
Comment (rudrakshi99) [inline Code/C++/primenumbers_till_n_nestedforloop.cpp:119]: Here also, remove the statement and mention O(n^2) only like this `Time Complexity : O(n^2)`.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

File: Code/C++/primenumbers_till_n_nestedforloop.cpp (hunk 3 above)
Comment (rohank2502) [reply Code/C++/primenumbers_till_n_nestedforloop.cpp:119]: ook

**Comment by rohank2502** [comment]
I have done suggested changed @rudrakshi99

**Comment by rudrakshi99** [comment]
Add the algorithm name to the list in the number theory folder.

**Review by rudrakshi99 — COMMENTED** [review]
Have you deleted the code ?

**Comment by rohank2502** [comment]
> 
> 
> Have you deleted the code ?

Yes first I added both in one PR but then i made seperate branch and created PR from both seperate branches Both PR has 1 question each with method 1

**Review by rudrakshi99 — APPROVED** [review]

**Review by rudrakshi99 — APPROVED** [review]

File: Code/C++/primenumbers_till_n_nestedforloop(Method_1).cpp [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+/*
+
+Time Complexity : O(N^2)
+
+*/
+/*
+
+Space Complexity :- O(1)
+*/
```
Comment (tarun26091999) [inline Code/C++/primenumbers_till_n_nestedforloop(Method_1).cpp:125]: remove unnecessary spaces.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
resolve the conflict.

**Pull request closed without merging**